### ISSUE TITLE
typo in examples.md

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/mean-share-and-balance#346]
For https://github.com/phetsims/qa/issues/1134

Under **Determine if the notepad is displaying the mean**  

You have a '1' instead of a tick mark for the first item listed under **Balance Point Screen**

<img width="374" alt="Screenshot 2024-08-14 at 7 26 50 PM" src="https://github.com/user-attachments/assets/489d5d15-6962-4d1a-9650-9e10015f88ba">

**Comment by Nancy-Salpepi** [comment]
Also, under Disable interaction with data values, consider changing the '*' to a '1' in: 
`meanShareAndBalance.balancePointScreen.model.sceneModel.soccerBalls.soccerBall*.enabledProperty`
to be consistent with the examples given for the other 3 screens and to make it a functional ID.

**Comment by KatieWoe** [comment]
Also fo "If meanFulcrumFixedProperty is false, the fulcrum is movable. Therefore, if meanValueProperty = meanPredictionFulcrumValueProperty, the sim is displaying the mean distance kicked." it may be relevant to note that the values don't necessary perfectly equal each other. The mean could be 3.6666666... and the correct prediction would be 3.7

**Comment by marlitas** [comment]
While discussing this with @amanda-phet I noticed that we weren't consistently using our mean rounding constant in the code. I cleaned that up above. Doesn't affect any sim behavior.

**Comment by amanda-phet** [comment]
Thanks for finding this typo, I fixed it! 

>Also, under Disable interaction with data values, consider changing the '*' to a '1' in:
meanShareAndBalance.balancePointScreen.model.sceneModel.soccerBalls.soccerBall*.enabledProperty
to be consistent with the examples given for the other 3 screens and to make it a functional ID.

I think this is standard for something with lots of numbered properties, so I don't point to a specific one. 

>it may be relevant to note that the values don't necessary perfectly equal each other. The mean could be 3.6666666... and the correct prediction would be 3.7

I updated the language to clarify that the meanValueProperty should be rounded to the nearest tenth. Thanks!
